src/components/AlphanumericInput/AlphanumericInput.svelte

```diff
@@ -3,13 +3,14 @@
   import { KEY } from '../../const'
 
   export let value: string
+  const normalizedValue = value.replace(' ', '')
```

Comment:
I don't like the idea of "making sure the dev is using it well" 🤔 If we start doing this it will be an endless job. You can then for example check that the value is in upper case and no symbol etc... If the dev is not using it the right way then it's his fault. I propose to remove that as we assume that the given prop `value` doesn't contain empty spaces

Reply:
✅